
syntax = "proto3";

package driverPackage;

service Driver {
    rpc launchDriver (launchData) returns (status);
}

message empty {} // empty message

message status {
    int32 code = 1;
    string msg = 2;
}

message launchData {
    string dirPath = 1;
    int32 numMappers = 2;
    int32 numReducers = 3;
    string ports = 4;
    int32 numClusters = 5;
    int32 iters = 6;
}